app/Http/Controllers/Api/ApiController.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Gladiator\Http\Controllers\Api;
+
+use League\Fractal\Manager;
+use League\Fractal\Resource\Item;
+use League\Fractal\Resource\Collection;
+use League\Fractal\Serializer\DataArraySerializer;
+use Gladiator\Http\Controllers\Controller as BaseController;
+
+class ApiController extends BaseController
+{
+    /**
+     * @var League\Fractal\Manager
+     */
+    protected $manager;
+
+    /**
+     * @var \League\Fractal\TransformerAbstract
+     */
+    protected $transformer;
+
+    /**
+     * Create new ApiController instance.
+     */
+    public function __construct()
+    {
+        $this->manager = new Manager;
+        $this->manager->setSerializer(new DataArraySerializer);
```

Comment:
I'm thinking it might be best to pull these out of the `__construct()` and place in the `transformer()` method so that the controller extending this ApiController doesn't have to always declare `parent::__construct()` in its `__construct()`...